test/integration/test_api_test_environment_certificate.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from exasol_integration_test_docker_environment.lib.docker import ContextDockerClient
+from exasol_integration_test_docker_environment.lib.test_environment.db_version import (
+    db_version_supports_custom_certificates,
+)
+from exasol_integration_test_docker_environment.test.get_test_container_content import (
+    get_test_container_content,
+)
+from exasol_integration_test_docker_environment.testing.api_test_environment import (
+    ApiTestEnvironment,
+)
+from exasol_integration_test_docker_environment.testing.api_test_environment_context_provider import (
+    build_api_context_provider_with_test_container,
+)
+from exasol_integration_test_docker_environment.testing.utils import (
+    check_db_version_from_env,
+)
+
+
+@pytest.fixture(scope="module")
+def environment_with_certificate(
+    api_isolation_module: ApiTestEnvironment,
+):
+    # Important: The default test environment name will be long for the certificate,
+    #            because host name length is limited to 63 characters.
+    #            Hence, we override the environment name here, we a shorter name.
+    api_isolation_module.name = "cert_test"
+    additional_parameter = {"create_certificates": True}
+
+    provider = build_api_context_provider_with_test_container(
+        api_isolation_module, get_test_container_content()
+    )
+    with provider(
+        None,
+        additional_parameter,
+    ) as db:
+        yield db
+
+
+@pytest.mark.skipif(
+    not db_version_supports_custom_certificates(check_db_version_from_env()),
+    reason="Database not supported",
```

Comment:
Value for `reason` and condition for `skipif` don't match semantically, IMHO.
If you explain the background to me, I can try to come up with a better name.

Reply:
Changed reason to "Database version does not support custom certificates.".

Reply:
:+1: